src/js/views/globals/search/patient-search_picklist.js

```diff
@@ -0,0 +1,261 @@
+
+import 'scss/modules/forms.scss';
+
+import intl from 'js/i18n';
+import hasAllText from 'js/utils/formatting/has-all-text';
+
+import Component from 'js/base/component';
+
+import 'js/components/picklist/picklist.scss';
+
+import InputFocusBehavior from 'js/behaviors/input-focus';
+import InputWatcherBehavior from 'js/behaviors/input-watcher';
+import PicklistBehavior from 'js/behaviors/picklist-transport';
+
+const CLASS_OPTIONS = [
+  'attr',
+  'canClear',
+  'childView',
+  'childViewEventPrefix',
+  'className',
+  'clearText',
+  'emptyView',
+  'emptyViewOptions',
+  'headingText',
+  'infoText',
+  'isSelectlist',
+  'lists',
+  'noResultsText',
+  'placeholderText',
+  'template',
+];
+
+const CLASS_OPTIONS_ITEM = [
+  'attr',
+  'getItemSearchText',
+  'itemClassName',
+  'itemTemplate',
+  'itemTemplateContext',
+];
+
+const attr = 'text';
+const canClear = false;
+const isSelectList = false;
+
+const PicklistEmpty = View.extend({
+  tagName: 'li',
+  className: 'picklist--no-results',
+  template: hbs`{{ noResultsText }}`,
+  serializeData() {
+    return this.options;
+  },
+});
+
+const PicklistItem = View.extend({
+  tagName: 'li',
+  itemTemplate: hbs`{{#if icon}}{{fa icon.type icon.icon classes=icon.classes}}{{/if}}<span>{{matchText text query}}</span>`,
+  itemClassName() {
+    const classNames = [];
+
+    if (this.model.get('isDisabled')) classNames.push('is-disabled');
+    if (this.model.get('hasDivider')) classNames.push('has-divider');
+
+    return classNames.join(' ');
+  },
+  className() {
+    const classNames = ['picklist__item', 'js-picklist-item', result(this, 'itemClassName', '')];
+
+    if (this.model === this.state.get('selected')) classNames.push('is-selected');
+
+    return classNames.join(' ');
+  },
+  triggers: {
+    'click': 'select',
+  },
+  preinitialize(options) {
+    this.mergeOptions(options, ['state', ...CLASS_OPTIONS_ITEM]);
+  },
+  onRender() {
+    this.searchText = this.getItemSearchText(this.model);
+  },
+  getItemSearchText(item) {
+    return this.$el.text();
+  },
+  itemTemplateContext: noop,
+  templateContext() {
+    return extend({
+      text: this.model.get(this.attr),
+      query: this.state.get('query'),
+    }, result(this, 'itemTemplateContext'));
+  },
+  getTemplate() {
+    return this.itemTemplate;
+  },
+});
+
+const Picklist = CollectionView.extend({
+  className: 'picklist__group',
+  tagName: 'li',
+  template: hbs`
+    <div class="patient-search__picklist-header">
+      <div class="patient-search__picklist-header-name">
+        <span>Patient</span>
+      </div>
+      <div class="patient-search__picklist-header-meta">
+        <div class="patient-search__picklist-header-dob">
+          <span>DOB</span>
+        </div>
+        {{#if showIdentifierHeader}}
+          <div class="patient-search__picklist-header-identifier">
+            <span>Identifier</span>
+          </div>
+        {{/if}}
+      </div>
+    </div>
+    <ul></ul>
```

Comment:
To get this to work, I copied over the entire picklist component. But this is where the update to it was made to get the header shown above the picklist items.

Still need to clean up this file and reduce it down to only what's needed.